tests/tck/features/VarLengthAcceptance.feature

```diff
@@ -65,28 +65,29 @@ Feature: VarLengthAcceptance
              (n011)-[:LIKES]->(n0110),
              (n011)-[:LIKES]->(n0111)
       """
-  @skip
+
   Scenario: Handling unbounded variable length match
     When executing query:
       """
       MATCH (a:A)
       MATCH (a)-[:LIKES*]->(c)
       RETURN c.name
+      ORDER BY c.name
```

Comment:
We should open a PR on the openCypher repo to add this fix - I should be able to do so today.